### PR TITLE
Add category transactions & user to budget app

### PR DESCRIPTION
**Pull request by imranhossainemi**
### Budget app Project  update

- Set up the repository and tools.
- Create models according to the ERD diagram. 
- Implement the authentication and authorization.
- Create the page to add a category.
- Create the home page.
- Create the page to add a transaction.
- Create the transactions page.
- Make sure my app is tested adequately.
- Deploy the project, and test for final details.
- Record a video for myproject.
- Create a good README and PR description.

### CODE REVIEW

File: app/views/budget_affairs/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    <%= form.label :name, class: "form-label mt-3" %>
+    <%= form.text_field :name, class: "form-control" %>
+
+    <%= form.label :amount, class: "form-label mt-3" %>
+    <%= form.number_field :amount, class: "form-control" %>
+
+    <%= form.submit "Add", class: "btn btn-success btn-lg mt-3 container-md" %>
+  <% end %>
+</section>
```
Comment (rotimiazeez) [inline app/views/budget_affairs/new.html.erb:25]: According to the project requirement, your app should have a Splash Screen: a simple page with the name of your app with links to the sign-up and log-in pages. Similar to the below design
Screenshot from 2023-07-15 11-11-46

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @imranhossainemi 👍🏽,
## Good job so far!
### To Highlight 👇🏽
### To Highlight 👇🏽
- Good project presentation video 📹. 
-  users can `Login` and `sign-up` on the app 💯
- The `home` page and `transaction` are functioning as required 👏🏾
- Good app UI design 🖼
-Tests are implemented and are passing ✔️
- App has been deployed and is available online 💪🏽 
- No Linter error ✔️ 
- Good PR description 👍🏽
- Good commit message(s) 😉
- Descriptive README 📖

You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: app/views/budget_affairs/new.html.erb (hunk 1 above)
Comment (imranhossainemi) [reply app/views/budget_affairs/new.html.erb:25]: @rotimiazeez It's alright there in user index page

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @imranhossainemi, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**